src/megameklab/com/ui/aerospace/AdvancedAeroStructureTab.java

```diff
@@ -175,23 +174,6 @@ public void refresh() {
 
     }
 
-    public void removeSystemCrits(int systemType) {
-
-        for (int loc = 0; loc < getJumpship().locations(); loc++) {
-            for (int slot = 0; slot < getJumpship().getNumberOfCriticals(loc); slot++) {
-                CriticalSlot cs = getJumpship().getCritical(loc, slot);
-
-                if ((cs == null) || (cs.getType() != CriticalSlot.TYPE_SYSTEM)) {
-                    continue;
-                }
-
-                if (cs.getIndex() == systemType) {
-                    getJumpship().setCritical(loc, slot, null);
-                }
-            }
-        }
-    }
-
```

Comment:
This method is a vestige of copying and pasting the mech structure tab as a starting point. It's not used for aerospace units (same with DS/SC below).